java/src/jmri/jmrit/beantable/ListedTableFrame.java

```diff
@@ -223,7 +223,7 @@ void buildMenus(final TabbedTableItem item) {
         JMenu fileMenu = new JMenu(rb.getString("MenuFile"));
         menuBar.add(fileMenu);
 
-        JMenuItem newItem = new JMenuItem("New Window");
+        JMenuItem newItem = new JMenuItem(rb.getString("MenuNewWindow"));
```

Comment:
The two localization changes in this file should be using Bundle.getMessage(...) instead of rb.getString(...)